ariadne/cardano/src/Ariadne/Cardano/Backend.hs

```diff
@@ -111,10 +118,17 @@ runCardanoNode protocolMagic bHandle addUs cardanoContextVar diffusionVar
         convertMode f diff =
             cardanoModeToRealMode $ f (hoistDiffusion realModeToCardanoMode cardanoModeToRealMode diff)
 
+        -- It's needed because 'bracketNodeResources' uses its own
+        -- logic to create a logging config, which differs from what
+        -- we do above. We want 'ncLoggerConfig' to be the same config
+        -- as the one passed to 'setupLogging'.
+        setProperLogConfig :: NodeResources __ -> NodeResources __
+        setProperLogConfig nr =
+            nr {nrContext = (nrContext nr) {ncLoggerConfig = loggerConfig}}
```

Comment:
Are there no lenses for `NodeResources` and `NodeContext`? Using them here would seem like a much cleaner solution.